testnets/upticktestnet/chain.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "upticktestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.uptickproject.com/",
+  "pretty_name": "Uptick",
+  "chain_id": "uptick_7000-2",
+  "bech32_prefix": "uptick",
+  "daemon_name": "uptickd",
+  "node_home": "$HOME/.uptickd",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "auptick",
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/UptickNetwork/uptick",
+    "recommended_version": "v0.2.6",
+    "compatible_versions": [
+      "v0.2.6"
+    ],
+    "cosmos_sdk_version": "0.46.5",
+    "tendermint_version": "0.34.23",
```

Comment:
oh, tendermin_version was changed for the consensus object (allows for tendermint of cometbft).
Also, if it's not too much trouble, could you please add this version data into the versions array?

Reply:
Check the fix